### ISSUE TITLE
a picture or two

### ISSUE THREAD

**Issue by tido-** [muesli/saloon#1]
Hi,

I came by because "duf" was mentioned in the Ubuntu Podcast and saw that you're quite active on Github and this project caught my attention, sounds interesting.
I would like to see a picture of it in action or a changing GIF would be nice. Also, I was wondering, what the Neopixel is for?
Last but not least, I thought to myself: Have I a display that fits? I had to search what yours are. Would be nice to amend the product number with: OLED 128x64

Cheers
Tido

**Comment by muesli** [comment]
Agreed, I need to do a better job in the marketing department :smiley:

Here's an example of Saloon running as a weather station in my office:
![image](https://user-images.githubusercontent.com/146378/120469279-6ca80100-c3a2-11eb-88cf-b070555aa6a7.png)

There's actually a whole variety of displays (a couple dozens at least) supported by the `u8g2` library. The Neopixel can be used for alerts/notifications: incoming messages, alarms, etc.

**Comment by tido-** [comment]
Thank you. Good idea with the notifications. The design of the housing is quite unique. Did you make it with a 3D printer?

**Comment by muesli** [comment]
Yeah, it's a modular & parametric case I've designed, which you can print on your 3D printer: https://github.com/muesli/modular-case. There are a bunch of modules for all kinds of boards, sensors and displays.
